packages/terra-clinical-item-display/src/ItemDisplay.jsx

```diff
@@ -36,25 +41,29 @@ const defaultProps = {
   text: '',
   textStyle: undefined,
   isTruncated: false,
+  isInvalid: false,
   icon: undefined,
 };
 
 const ItemDisplay = ({
     text,
     textStyle,
     isTruncated,
+    isInvalid,
     icon,
     ...customProps
   }) => {
   const textClassNames = cx([
     'text',
+    { 'is-invalid': isInvalid },
     { 'is-truncated': isTruncated },
     { 'strike-through': textStyle === 'strikeThrough', [`${textStyle}`]: textStyle },
   ]);
 
   let displayIcon;
   if (icon) {
-    displayIcon = <div className={cx('inline-icon')}>{icon}</div>;
+    const iconClasses = cx('inline-icon', { 'is-invalid': isInvalid });
+    displayIcon = <div className={iconClasses}>{icon}</div>;
   }
 
   return (
```

Comment:
I feel like we should place the 'is-invalid' class on the Item-Display div instead of the text and icon divs. Is there a reason we apply it twice?

Reply:
Yeah I considered that and went with this approach since it was already being applied directly to the text instead of surrounding div. I agree with you though. I'll update this.